module.nix

```diff
@@ -125,6 +141,20 @@ in
     # authentik server
     (mkIf config.services.authentik.enable (let
       cfg = config.services.authentik;
+
+      # Passed to each service and to the `ak` wrapper using `systemd-run(1)`
+      serviceDefaults = {
+        DynamicUser = true;
+        User = "authentik";
+        EnvironmentFile = mkIf (cfg.environmentFile != null) [ cfg.environmentFile ];
+      };
+      akOptions = flatten (mapAttrsToList
+        (k: vs: map (v: "-p ${k}=${if isBool v then boolToString v else toString v}") (toList vs))
```

Comment:
```suggestion
      akOptions = flatten (mapAttrsToList
        # map properties from the authentik serviceConfig to command line parameters passed to systemd-run(1)
        (k: vs: map (v: "--property ${k}=${if isBool v then boolToString v else toString v}") (toList vs))
```

Reply:
Good point. Added another sentence to explain why we're doing this as well :)